eland/tests/dataframe/test_sample_pytest.py

```diff
@@ -0,0 +1,55 @@
+# Copyright 2020 Elasticsearch BV
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# File called _pytest for PyCharm compatibility
+
+from eland.tests.common import TestData
+from eland.tests.common import assert_pandas_eland_frame_equal
+
+
+class TestDataFrameSample(TestData):
```

Comment:
Eventually will need to add a test case that calls `.sample()` and then other operations such a `.head()`, `.agg()`, `.shape`, etc

Reply:
Thank you so much for your thorough review of this PR, I learn a lot with each comment you made. My doubt with the test is how do I assert is working, I mean what assertion should I check. I already fix some minor issues and believe I can solve the others early next week.

Reply:
The combination asserts will probably be easier after implementing `random_state`. Mostly want to verify that we can add additional queries to our .sample() calls without pulling data from ES

Reply:
As far as checking whether `.sample()` itself is working you could test that calling `.sample(10)` twice gives you two different sets of rows :)